crates/benchmarks/benches/modules/helpers/rot_surface.rs

```diff
@@ -0,0 +1,276 @@
+use pecos_core::IndexableElement;
+use pecos_qsims::CliffordSimulator;
+use std::collections::HashMap;
+
+type Layout = HashMap<usize, (usize, usize)>;
+
+// TODO: Create a simplified function that generates the list of gates and qubits in the right
+//  space and time ordering.
+
+#[allow(dead_code)]
+struct RotSurface {
+    distance: usize,
+    width: usize,
+    height: usize,
+    data_qubit_ids: Vec<usize>,
```

Comment:
Nit: What are the usizes supposed to represent in this type? Are they related to the width and height in the `RotSurface` struct? It may help to either use a named struct or in the case that the tuple is easier to use, oruse type aliases for the different usizes so the interpretation is easy to grok from reading the type. Or alternately document what the type means in a comment. Also, I noticed that though the `layout` field in `RotSurface` is the same type, this type alias is not used.

Reply:
This is connected with pecos-qec not being ready yet. Will fix.